lib/resources/package.rb

```diff
@@ -44,6 +44,8 @@ def initialize(package_name, opts = {}) # rubocop:disable Metrics/AbcSize
         @pkgman = SolarisPkg.new(inspec)
       elsif ['hpux'].include?(os[:family])
         @pkgman = HpuxPkg.new(inspec)
+      elsif os.alpine?
```

Comment:
I would use `['alpine'].include?(os[:name])`, we do not need a new helper for alpine